### PR TITLE
Refactoring P-384 implementation for consistency

### PR DESCRIPTION
**Pull request by dkostic**
### Issues:
N/A

### Description of changes: 
Refactoring P-384 implementation for consistency (removing `fiat` prefix from functions, enforcing 80 characters per line, etc.)

### Call-outs:
N/A

### Testing:
How is this change tested (unit tests, fuzz tests, etc.)? Are there any testing steps to be verified by the reviewer?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: crypto/fipsmodule/ec/p384.c [hunk 1]
```diff
@@ -500,11 +505,12 @@ static void ec_GFp_nistp384_dbl(const EC_GROUP *group, EC_RAW_POINT *r,
 // The calls to from/to_generic are needed for the case
 // when BORINGSSL_HAS_UINT128 is undefined, i.e. p384_32.h fiat code is used;
 // while OPENSSL_64_BIT is defined, i.e. BN_ULONG is uint64_t
-static void ec_GFp_nistp384_mont_felem_to_bytes(const EC_GROUP *group, uint8_t *out,
-                                         size_t *out_len, const EC_FELEM *in) {
+static void ec_GFp_nistp384_mont_felem_to_bytes(
+  const EC_GROUP *group,uint8_t *out, size_t *out_len, const EC_FELEM *in) {
```
Comment (DominicDams) [inline crypto/fipsmodule/ec/p384.c:509]: Nit: missing space
```
   const EC_GROUP *group, uint8_t *out, size_t *out_len, const EC_FELEM *in) {
```

File: crypto/fipsmodule/ec/p384.c [hunk 2]
```diff
@@ -933,12 +943,13 @@ static void ec_GFp_nistp384_point_mul_base(const EC_GROUP *group,
 // Both scalar products are computed by the same "textbook" wNAF method,
 // with w = 7 for g_scalar and w = 5 for p_scalar.
 // For the base point G product we use the first sub-table of the precomputed
-// table |fiat_p384_g_pre_comp| from p384_table.h file, while for P we generate
-// the |p_pre_comp| table on-the-fly. The tables hold the first 64 odd multiples
+// table |p384_g_pre_comp| from p384_table.h file, while for P we generate
+// |p_pre_comp| table on-the-fly. The tables hold the first 64 odd multiples
```
Comment (DominicDams) [inline crypto/fipsmodule/ec/p384.c:947]: Nit: Add back the or remove the word table

**Review by DominicDams — DISMISSED** [review]

File: third_party/fiat/p384_64.h [hunk 3]
```diff
@@ -26,15 +26,6 @@ __extension__ typedef __uint128_t fiat_p384_uint128;
 #error "This code only works on a two's complement system"
 #endif
 
-#if !defined(FIAT_P384_NO_ASM) && (defined(__GNUC__) || defined(__clang__))
```
Comment (nebeid) [inline third_party/fiat/p384_64.h:29]: These definitions were not used anyway as stated on l.133, right?

Comment (dkostic) [reply third_party/fiat/p384_64.h:29]: correct

**Review by nebeid — APPROVED** [review]

**Review by torben-hansen — APPROVED** [review]
lgtm

**Review by nebeid — APPROVED** [review]

**Pull request merged**